crutest/src/main.rs

```diff
@@ -124,26 +124,37 @@ enum Workload {
 #[clap(name = "client", term_width = 80)]
 #[clap(about = "A Crucible upstairs test client", long_about = None)]
 pub struct Opt {
+    // TLS options
```

Comment:
These options were put in alphabetical order.

The only new option here is the read_only.  The rest are just moved around.